apps/charge-accounts-service/src/operators/config/configuration.ts

```diff
@@ -0,0 +1,16 @@
+export default () => ({
```

Comment:
@MusabShakeel576 To avoid duplications, I suggest utilizing the configuration settings found in `/accounts-service/paymaster/config/configuration`.